plinth/modules/users/__init__.py

```diff
@@ -84,3 +84,11 @@ def _diagnose_ldap_entry(search_item):
 
     return [_('Check LDAP entry "{search_item}"')
             .format(search_item=search_item), result]
+
```

Comment:
The idea is that each module creates its own ldap group during installation and removes it during uninstallation. I think these utility functions are better placed in action_utils.

Reply:
Also, can you do this one for one app, like Deluge for user group 'bittorrent' so that it can be tested easily?